### PR TITLE
Add h5utils recipe

### PR DESCRIPTION
**Pull request by ChristopherHogan**

### CODE REVIEW

File: recipes/h5utils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - libpng
+    - hdf5
+
+test:
+  commands:
+    - test -f ${PREFIX}/bin/h5topng
+
+about:
+  home: http:/github.com/stevengj/h5utils
+  license: MIT
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/meta.yaml]: Fixed.

File: recipes/h5utils/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+export CXXFLAGS="-I${CFLAGS} ${CXXFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+
+./configure --prefix=${PREFIX}
+make
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/build.sh]: Unfortunately no.

File: recipes/h5utils/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+export CXXFLAGS="-I${CFLAGS} ${CXXFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+
+./configure --prefix=${PREFIX}
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/build.sh]: Fixed.

File: recipes/h5utils/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+    - libpng
+    - hdf5
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/meta.yaml]: Fixed.

File: recipes/h5utils/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  run:
+    - libpng
+    - hdf5
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/meta.yaml]: Fixed.

File: recipes/h5utils/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
+    - libpng
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/meta.yaml]: Fixed.

File: recipes/h5utils/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    - hdf5
+  run:
+    - libpng
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/meta.yaml]: Fixed.

File: recipes/h5utils/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/meta.yaml]: Fixed.

File: recipes/h5utils/build.sh [hunk 9]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+export CXXFLAGS="-I${CFLAGS} ${CXXFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
```
[the remark this answers is not in the document]
Comment (ChristopherHogan) [reply recipes/h5utils/build.sh]: Fixed.

File: recipes/h5utils/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libpng >=1.6.32,<1.6.35
+    - hdf5 1.10.1
```
Comment (jakirkham) [inline recipes/h5utils/meta.yaml]: Please add `zlib 1.2.11` as it seems to use it.

File: recipes/h5utils/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - libpng >=1.6.32,<1.6.35
+    - hdf5 1.10.1
```
Comment (jakirkham) [inline recipes/h5utils/meta.yaml]: Please add `zlib 1.2.11` as it seems to use it.

File: recipes/h5utils/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  commands:
+    - ${PREFIX}/bin/h5topng -h
```
Comment (jakirkham) [inline recipes/h5utils/meta.yaml]: Please drop `${PREFIX}/bin/` as this will be first on `$PATH` anyways.

Comment (jakirkham) [inline recipes/h5utils/meta.yaml]: Also please add tests (help or otherwise) for...

* `h5totxt`
* `h5fromtxt`
* `h5tovtk`

**Comment by jakirkham** [comment]
Thanks for fixes.

Mentioned a couple more things. Once those are done, think we will be ready to merge.

**Comment by ChristopherHogan** [comment]
I've addressed all comments.

File: recipes/h5utils/meta.yaml (hunk 10 above)
Comment (jakirkham) [reply recipes/h5utils/meta.yaml]: note: Done below

**Comment by jakirkham** [comment]
@conda-forge-admin, please relint.

**Comment by jakirkham** [comment]
Thanks @ChristopherHogan. It's in.

FYI after this converts, an invite will be sent to you to join conda-forge, which will provide you write permissions on this feedstock. Please read the [docs]( https://conda-forge.org/docs/ ) and the contributing guides in the feedstock to learn more about how best to maintain it going forward. Also please feel free to ask any questions that may come to mind. Welcome to conda-forge.

**Pull request merged**